### ISSUE TITLE
unit tests for API

### ISSUE THREAD

**Issue by daveneti** [corehunter/corehunter3#75]
@hdbeukel I don't see any unit tests for the API class, is this correct or am I missing something. I am going to add unit test for the methods I created in API. I will look at yours given time.

**Comment by hdbeukel** [comment]
You are right, I don’t believe the API is tested at the Java side. Of course the R package uses the API and is fully tested but it might be interesting to add Java tests as well.

**Comment by daveneti** [comment]
ok, I have found and fixed the bug that caused me to look for the API . I will merge this to develop branch now, but I have not added all the unit tests yet.
